go/vt/sqlparser/ast_test.go

```diff
@@ -697,7 +697,7 @@ func TestColumns_FindColumn(t *testing.T) {
 	}
 }
 
-func TestSplitStatements(t *testing.T) {
+func TestParseMultipleIgnoreEmpty(t *testing.T) {
```

Comment:
nitpicking: this doesn't feel like an `ast_test`

Reply:
I left it there, just so that is clear what the difference in outputs are for the existing tests, but I can move them.

Reply:
Done